.travis.yml

```diff
@@ -7,25 +7,25 @@ addons:
     packages:
       - google-chrome-stable
 language: node_js
+cache:
+  yarn: true
 node_js:
   - "8"
 cache:
   - node_modules
 before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
-  - npm config set package-lock false
-  - npm i -g npm@^5.7.1
+  - npm i -g yarn
 install:
-  - npm install
-  - npm install mgit2 lerna
+  - yarn add mgit2 --ignore-workspace-root-check
   - mgit bootstrap --resolver-url-template="https://github.com/\${ path }.git"
-  - lerna bootstrap
+  - yarn
 script:
-  - npm t -- --reporter=dots
-  - npm run docs:api -- --validate-only
+  - yarn run test --reporter=dots
+  - yarn run docs:api --validate-only
```

Comment:
Also no need to `run`.

Reply:
Actually, I made a couple of suggestions here in this PR, but it'll be easier to change this automatically in all places. `s/yarn run/yarn` seems to work for me.